shell/browser/api/atom_api_web_request_ns.cc

```diff
@@ -99,6 +100,33 @@ bool MatchesFilterCondition(extensions::WebRequestInfo* info,
   return false;
 }
 
+// Convert HttpResponseHeaders to V8.
+//
+// Note that while we already have converters for HttpResponseHeaders, we can
+// not use it because it lowercases the header keys, while the webRequest has
+// to pass the original keys.
+v8::Local<v8::Value> HttpResponseHeadersToV8(
+    net::HttpResponseHeaders* headers) {
+  base::DictionaryValue response_headers;
+  if (headers) {
+    size_t iter = 0;
+    std::string key;
+    std::string value;
+    while (headers->EnumerateHeaderLines(&iter, &key, &value)) {
+      if (response_headers.FindKey(key)) {
+        base::ListValue* values = nullptr;
+        if (response_headers.GetList(key, &values))
+          values->AppendString(value);
```

Comment:
AppendString is deprecated, we could use GetList().emplace_back() instead. _(minor)_ Also we can avoid the second lookup if we use FindListKey()

```c++
Value* values = response_headers.FindListKey(key);
if (values != nullptr) {
  values->GetList().emplace_back(value);
}
```
What's the difference between `FindKey()` + `GetList()` and `FindListKey()`?

Reply:
They should do the same work.